.github/workflows/scorecards.yml

```diff
@@ -1,6 +1,7 @@
 name: "OSSF Scorecard"
 on:
   branch_protection_rule:
+  workflow_dispatch:
```

Comment:
Would it be possible to add a brief description of the change on the body of the commit message?